src/main/java/org/sigmah/shared/dto/pivot/content/PivotTableData.java

```diff
@@ -59,7 +59,7 @@ public PivotTableData(List<Dimension> rowDimensions, List<Dimension> columnDimen
 		rootRow = new Axis();
 		rootColumn = new Axis();
 	}
-
+    
```

Comment:
@manishvishnoi2 : Redundant space. Needs to be removed before the pull request can be merged into the codebase.